model_analyzer/perf_analyzer/perf_analyzer.py

```diff
@@ -187,14 +188,23 @@ def get_records(self):
         """
```

Comment:
Should this be renamed to get_perf_records?

Reply:
Done